service/src/test/java/uk/nhs/adaptors/gp2gp/common/service/ConfidentialityServiceTest.java

```diff
@@ -0,0 +1,92 @@
+package uk.nhs.adaptors.gp2gp.common.service;
+
+import org.hl7.fhir.dstu3.model.Coding;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.nhs.adaptors.gp2gp.common.configuration.RedactionsContext;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(MockitoExtension.class)
+public class ConfidentialityServiceTest {
+    private static final List<Coding> NO_META_SECURITY = List.of(new Coding());
+    private static final List<Coding> NOPAT_META_SECURITY = List.of(
+        new Coding(
+            "http://hl7.org/fhir/v3/ActCode",
+            "NOPAT",
+            "no disclosure to patient, family or caregivers without attending provider's authorization"
+        )
+    );
+    private static final List<Coding> NON_NOPAT_META_SECURITY = List.of(
+        new Coding(
+            "http://hl7.org/fhir/v3/ActCode",
+            "NON-NOPAT",
+            "random disclosure to patient, family or caregivers without attending provider's authorization"
+        )
+    );
+    public static final String NON_REDACTION_INTERACTION_ID = "RCMR_IN030000UK06";
+    public static final String REDACTION_INTERACTION_ID = "RCMR_IN030000UK07";
```

Comment:
Would it be worth having these within some externalised test utility, I can foresee this being handy within other tests within the future!

Although, arguably, my suggestion may be going against YAGNI 😆

Reply:
Have moved these into the RedactionsContext